ivy/functional/frontends/torch/tensor.py

```diff
@@ -774,6 +774,12 @@ def is_cuda(self):
     def is_meta(self):
         return "meta" in ivy.dev(self.ivy_array)
 
+    @with_unsupported_dtypes(
+        {"2.1.0 and below": ("uint16", "bfloat16", "bool")}, "torch"
```

Comment:
```suggestion
        {"2.1.0 and below": ("uint16", "bool")}, "torch"
```
I think bfloat16 is supported for this function